lib/internal/http2/core.js

```diff
@@ -848,7 +863,7 @@ class ServerHttp2Stream extends Http2Stream {
       options.readable = !options.endStream;
       const stream = new ServerHttp2Stream(session, ret, options);
       streams.set(ret, stream);
-      process.nextTick(callback, stream, headers, 0);
+      process.nextTick(callback, null, stream, headers, 0);
```

Comment:
I assume the `null` here is to make it match the typical Node.js callback `(err, ...args)` pattern? If so, I do not believe that's necessary. The objects involved here are event emitters that will emit `error` events when errors occur.